### ISSUE TITLE
Placeholder API extension NullPointerException

### ISSUE THREAD

**Issue by FPSUsername** [SkinsRestorer/SkinsRestorerX#593]
## Describe the bug
Placeholder API reports a NullPointerException when loading the SkinsRestorer expansion.

## How can we reproduce what you got?
Install [SkinsRestorer](https://www.spigotmc.org/resources/skinsrestorer.2124/) and [PlaceholderAPI](https://www.spigotmc.org/resources/placeholderapi.6245/) plugins

Install the extension through the papi command:
1. Start the server and download the papi extension (`/papi ecloud download skinsrestorer`)
2. Reload papi (`/papi reload`) or restart the server

Or install the extension manually:
1. Download it from [eCloud](https://api.extendedclip.com/expansions/skinsrestorer/) and place it in `plugins/PlaceholderAPI/expansions`
2. Start the server

**Expected behavior**
Papi should load the expansion without errors

## Server information
* SkinsRestorer: 1.14.1.1
* Placeholder API: 2.10.10
* [PurPur](purpur.pl3x.net/) version: 1.17.1 build 1265
* Java: [Adopt OpenJDK 16 with OpenJ9](https://adoptopenjdk.net/?variant=openjdk16&jvmVariant=openj9)
* plugin list: https://hastebin.app/60ec6ad051a8e5001ddfe910
* Minecraft Launcher: Default
* Server Launcher: [Fork](https://fork.gg)

## Logs
Full log: https://hastebin.app/60ec68e451a8e5001ddfe90e
Partial log (error): https://hastebin.app/60ec690151a8e5001ddfe90f

**Comment by xknat** [comment]
Are you using our example? If so what version?

**Comment by xknat** [comment]
@FPSUsername Any update on this?

**Comment by FPSUsername** [comment]
I don't think I'm using the papi methods anywhere for this plugin, I checked my plugins (such as TabList), so technically I don't need this expansion.
This error log just seems to happen on my server. I did not test it on an empty fresh server though.

**Comment by xknat** [comment]
backend is in bungee mode?
if so, it has no database..

**Comment by FPSUsername** [comment]
I do not use bungee/waterfall, just a single standalone server.

**Comment by xknat** [comment]
java.lang.NullPointerException: The expansion version is null!
can you delete it and manual download the latest version?

**Comment by FPSUsername** [comment]
Tried that, but didn't work. I downloaded it manually from eCloud and tried it as well again with the papi command.

**Comment by xknat** [comment]
TY, i will look into this.
DEV notes: https://github.com/SkinsRestorer/SRPlaceholderAPIExpansion/commit/1e6244c54f37c50e66ec26550b3e9b65f7007b72#diff-6b392383fd4d6674d56d04bb6bac42b6a0e20d002628763b649853b8cd4250ffL46-R36

**Comment by xknat** [comment]
Hey there, can any one let me know if this issue is still there?

**Comment by FPSUsername** [comment]
It still happens when I install the extension using the papi command.

```
[11:43:46] [Server thread/ERROR]: [PlaceholderAPI] Failed to load expansion class SkinsRestorerExpansion - One of its properties is null which is not allowed!
[11:43:46] [Server thread/ERROR]: [PlaceholderAPI] 
java.lang.NullPointerException: The expansion version is null!
	at java.util.Objects.requireNonNull(Objects.java:233) ~[?:?]
	at me.clip.placeholderapi.expansion.manager.LocalExpansionManager.register(LocalExpansionManager.java:166) ~[PlaceholderAPI-2.10.10.jar:?]
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197) ~[?:?]
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179) ~[?:?]
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625) ~[?:?]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) ~[?:?]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) ~[?:?]
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913) ~[?:?]
	at java.util.stream.ReduceOps$5.evaluateSequential(ReduceOps.java:257) ~[?:?]
	at java.util.stream.ReduceOps$5.evaluateSequential(ReduceOps.java:248) ~[?:?]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?]
	at java.util.stream.ReferencePipeline.count(ReferencePipeline.java:709) ~[?:?]
	at me.clip.placeholderapi.expansion.manager.LocalExpansionManager.lambda$registerAll$3(LocalExpansionManager.java:327) ~[PlaceholderAPI-2.10.10.jar:?]
	at me.clip.placeholderapi.util.Futures.lambda$null$0(Futures.java:46) ~[PlaceholderAPI-2.10.10.jar:?]
	at org.bukkit.craftbukkit.v1_17_R1.scheduler.CraftTask.run(CraftTask.java:101) ~[patched_1.17.1.jar:git-Purpur-1311]
	at org.bukkit.craftbukkit.v1_17_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:483) ~[patched_1.17.1.jar:git-Purpur-1311]
	at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1565) ~[patched_1.17.1.jar:git-Purpur-1311]
	at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:492) ~[patched_1.17.1.jar:git-Purpur-1311]
	at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1480) ~[patched_1.17.1.jar:git-Purpur-1311]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1270) ~[patched_1.17.1.jar:git-Purpur-1311]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:319) ~[patched_1.17.1.jar:git-Purpur-1311]
	at java.lang.Thread.run(Thread.java:831) [?:?]
```

**Comment by xknat** [comment]
@FPSUsername can you mention me on our discord server, got a possible fix

**Comment by xknat** [comment]
fixed 1.4 -> with https://api.extendedclip.com/expansions/skinsrestorer/versions/skinsrestorer-14/
